src/hardware/setup.rs

```diff
@@ -1049,11 +1052,53 @@ pub fn setup(
             )),
         ];
 
+        let config = hal::spi::Config::new(hal::spi::Mode {
+            polarity: hal::spi::Polarity::IdleHigh,
```

Comment:
Wrong mode. CPOL=0 is IdleLow.

![image](https://user-images.githubusercontent.com/1338946/191207245-711a33f8-ee4c-471e-a3cf-ab1ad0d8bb4c.png)

![image](https://user-images.githubusercontent.com/1338946/191208102-d530420b-b040-4b2a-a22e-dad6f275361e.png)

![image](https://user-images.githubusercontent.com/1338946/191211605-7b0c5e49-287d-4256-a0c9-7cc7bba6463f.png)